[2024-03-16T03:00:15.497+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-03-16T02:00:00+00:00 [queued]>
[2024-03-16T03:00:15.504+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-03-16T02:00:00+00:00 [queued]>
[2024-03-16T03:00:15.504+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-16T03:00:15.521+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_data_to_postgres> on 2024-03-16 02:00:00+00:00
[2024-03-16T03:00:15.530+0000] {standard_task_runner.py:60} INFO - Started process 1148 to run task
[2024-03-16T03:00:15.532+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'load_data_to_postgres', 'scheduled__2024-03-16T02:00:00+00:00', '--job-id', '486', '--raw', '--subdir', 'DAGS_FOLDER/weather_api__dag.py', '--cfg-path', '/tmp/tmp7aqg0md0']
[2024-03-16T03:00:15.533+0000] {standard_task_runner.py:88} INFO - Job 486: Subtask load_data_to_postgres
[2024-03-16T03:00:15.574+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-03-16T02:00:00+00:00 [running]> on host a084c6a0d68a
[2024-03-16T03:00:15.661+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='kan@odds.team' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='load_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-03-16T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-16T02:00:00+00:00'
[2024-03-16T03:00:15.670+0000] {logging_mixin.py:188} INFO - /opt/***/dags/weather_data_2024-03-16T02:00:00+00:00.json
[2024-03-16T03:00:15.671+0000] {warnings.py:109} WARNING - /opt/***/dags/weather_api__dag.py:88: AirflowProviderDeprecationWarning: The "schema" arg has been renamed to "database" as it contained the database name.Please use "database" to set the database name.
  pg_hook = PostgresHook(

[2024-03-16T03:00:15.677+0000] {base.py:83} INFO - Using connection ID 'my_***_conn' for task execution.
[2024-03-16T03:00:15.692+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-16T03:00:15.699+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=load_data_to_***, execution_date=20240316T020000, start_date=20240316T030015, end_date=20240316T030015
[2024-03-16T03:00:15.744+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-16T03:00:15.761+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
